module/js/integrations/ChrisPremades.js

```diff
@@ -0,0 +1,246 @@
+import {IntegrationBase} from "./IntegrationBase.js";
+import {SharedConsts} from "../../shared/SharedConsts.js";
+
+// Cheat and pretend we're not always overriding, as our patches are temporary/highly contextual
+const _LIBWRAPPER_TYPE_TEMP = "MIXED";
```

Comment:
should really be `"OVERRIDE"`, but our patches here are put in place for for the duration of one call and then removed. To prevent conflicts with modules that _actually_ patch the same methods, we declare our patches as "MIXED"